src/test/scala/mesosphere/marathon/integration/AppDeployIntegrationTest.scala

```diff
@@ -30,6 +30,52 @@ class AppDeployIntegrationTest
   //clean up state before running the test case
   before(cleanUp())
 
+  // FIXME (gkleiman): Docker tests only work when the integration tests are run on Linux using bin/run-tests.sh
+  if (sys.env.contains("BUILD_ID")) {
+    test("deploy a simple Docker app") {
+      Given("a new Docker app")
+      val app = AppDefinition(
+        id = testBasePath / "dockerapp",
+        cmd = Some("true"),
```

Comment:
this will immediately stop the app and restart it -- is that really what we want here?